lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/file/Files.scala

```diff
@@ -0,0 +1,24 @@
+package org.enso.projectmanager.infrastructure.file
+
+import java.io.File
+
+object Files {
+
+  private val HOME_DIR = "~"
```

Comment:
The title of the PR is "... should handle relative paths" - I was expecting something different here then! This is not handling relative paths (which start with `./` and would use `System.getProperty("user.dir")`), but `~` as a reference to home directory!